src/Dfc.CourseDirectory.Core/DataManagement/ParsedCsvCourseRow.cs

```diff
@@ -52,9 +52,20 @@ public static ParsedCsvCourseRow FromCsvCourseRow(CsvCourseRow row, IEnumerable<
             _ => (CourseAttendancePattern?)null
         };
 
+        public static string ResolveAttendancePattern(CourseAttendancePattern value) => value switch
```

Comment:
I think we should use a prefix other than Resolve for these added methods that map to a string. The `*Resolved` fields are for the strongly-typed properties that have come _from_ a string. Perhaps `Map` (so `MapAttendancePattern`, `MapCost` etc)?

Reply:
good shout, changed.